src/types/UiaTextRangeBase.cpp

```diff
@@ -1318,6 +1336,15 @@ const Viewport UiaTextRangeBase::_getBufferSize() const noexcept
     return Viewport::FromDimensions({ 0, 0 }, width, height);
 }
 
+const til::point UiaTextRangeBase::_getDocumentEnd() const noexcept
+{
+    const auto optBufferSize{ _getBufferSize() };
+    const auto& buffer{ _pData->GetTextBuffer() };
+    const auto lastCharPos{ buffer.GetLastNonSpaceCharacter(optBufferSize) };
+    const auto cursorPos{ buffer.GetCursor().GetPosition() };
+    return { optBufferSize.Left(), std::max(lastCharPos.Y, cursorPos.Y) + 1 };

```

Comment:
isn't `optBufferSize.Left()` always 0?

Reply:
yeah. I've been trying to use `.Left()` instead of `0` hoping that it would make things more clear. But idk if there's a real benefit to that. Should I change it?